src/cascadia/Remoting/WindowManager.cpp

```diff
@@ -551,6 +552,19 @@ namespace winrt::Microsoft::Terminal::Remoting::implementation
         _peasant.RequestHideTrayIcon();
     }
 
+    // Method Description:
+    // - Ask the monarch to quit all windows.
+    // Arguments:
+    // - <none>
+    // Return Value:
+    // - <none>
+    winrt::fire_and_forget WindowManager::RequestQuitAll()
+    {
+        auto strongThis{ get_strong() };

```

Comment:
What's the `strongThis` for?

Reply:
I think I just copied from the above `RequestHideTrayIcon` and I assumed it was there for an important reason. It can probably be removed?

Reply:
Ah, found something on it: https://github.com/microsoft/terminal/pull/10938#issuecomment-899837721